versioned_docs/version-8.6/self-managed/identity/deployment/configuration-variables.md

```diff
@@ -27,6 +27,20 @@ As a Spring Boot application, Identity supports any standard
 | `KEYCLOAK_SETUP_CLIENT_ID`           | The client to use for authentication during setup of the provided Keycloak.         | admin-cli                                                                                                                                                                |
 | `KEYCLOAK_URL`                       | The URL of the Keycloak instance to use.                                            | http://localhost:18080/auth                                                                                                                                              |
 
+## Camunda Identity Configuration
+
```

Comment:
Add a small section that explains that these properties are used to configure the identity connection for each camunda component